crates/kubelet/Cargo.toml

```diff
@@ -14,7 +14,6 @@ edition = "2018"
 license-file = "../../LICENSE"
 description = "A Kubernetes kubelet implementation in Rust"
 repository = "https://github.com/deislabs/krustlet"
-readme = "README.md"
```

Comment:
#232 introduced this field in other crates as well - should it be removed from there too?

Reply:
We aren't publishing them, so it isn't a big deal, but I'll go remove them

Reply:
Actually, just checked. It does need to be there in oci-distribution and none of the rest have them